src/content/app/tools/blast/components/blast-app-bar/BlastAppBar.tsx

```diff
@@ -95,15 +95,15 @@ const BlastAppBar = () => {
 
   const speciesTabs = areSpeciesEnabled
     ? speciesList.map((species, index) => (
-        <SpeciesLozenge
+        <SelectedSpecies
           key={index}
-          theme="blue"
+          disabled={false}
```

Comment:
`disabled` is optional so this might not be necessary

Reply:
✅